### PR TITLE
Add Hijri calendar example

### LINKED ISSUE #714: Support for moment-hijri
Is there a plan to support hijri dates?

if not, could you please guide me through making one on my own?

### PR DESCRIPTION
**Pull request by alaatm**
This PR closes #714

### CODE REVIEW

File: docs/pages/localization/hijri.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Ad from '_shared/Ad';
```
Comment (dmtrKovalenko) [inline docs/pages/localization/hijri.mdx]: I think there is no way to make additional example for hijiri. We can make same page for both. 
Just title it like `Additional calendar systems` or smth. Thanks for your work

File: docs/pages/localization/hijri.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import Example from '_shared/Example'
+import PageMeta from '_shared/PageMeta'
+import * as HijriExample from './Hijri.example.jsx'
+
+<PageMeta title="Jalaali localization - @material-ui/pickers" />
```
Comment (dmtrKovalenko) [inline docs/pages/localization/hijri.mdx]: Poininting to Jalaali

**Comment by alaatm** [comment]
Moved both jalaali and hijri in single page.

File: docs/pages/localization/calendar-systems.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+You can use the examples below. It is recommended that you change the font.
+
+<Example source={PersianExample} />
+
+<hr/>
+
+## Hijri calendar system
+
+Make sure that you have read the [right to left section](https://material-ui-next.com/guides/right-to-left/) of the
```
Comment (dmtrKovalenko) [inline docs/pages/localization/calendar-systems.mdx]: Duplicated paragraph

File: docs/pages/localization/calendar-systems.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import Ad from '_shared/Ad';
+import Example from '_shared/Example'
+import PageMeta from '_shared/PageMeta'
+import * as PersianExample from './Persian.example.jsx'
+import * as HijriExample from './Hijri.example.jsx'
+
+<PageMeta title="Additional Calendar Systems - @material-ui/pickers" />
```
Comment (dmtrKovalenko) [inline docs/pages/localization/calendar-systems.mdx]: Please start the page with a short description. Like that you will need to install additional package. And setup right-to-left

File: docs/pages/localization/calendar-systems.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## Persian calendar system
+
+Make sure that you have read the [right to left section](https://material-ui-next.com/guides/right-to-left/) of the
+material-ui documentation page before proceeding.
+
+<Ad />
+
+You will also need to install the `@date-io/jalaali` package from npm.
```
Comment (dmtrKovalenko) [inline docs/pages/localization/calendar-systems.mdx]: Replace you will also with Install package. Like you have done for hijiri

**Comment by dmtrKovalenko** [comment]
No more comments. Waiting CI to pas. Sorry for requesting changes last time

**Comment by alaatm** [comment]
> No more comments. Waiting CI to pas. Sorry for requesting changes last time

No problem :) Thanks for your work!

**Comment by dmtrKovalenko** [comment]
Ohh. Just wired thing with headers and displaying https://share.icloud.com/photos/0ElVl0dcKx2ifbHWT0iht5Jhw

**Comment by alaatm** [comment]
> Ohh. Just wired thing with headers and displaying https://share.icloud.com/photos/0ElVl0dcKx2ifbHWT0iht5Jhw

I think headers look better now

**Pull request merged**